activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+*   Properly support `reverse_order` on relations with `nulls_first` or `nulls_last` calls
```

Comment:
`nulls_first` and `nulls_last` aren't public API, so this doesn't need a changelog entry.

Reply:
Thanks! Just rebased and removed it.